backend/db/company_jobs.py

```diff
@@ -0,0 +1,18 @@
+from sqlmodel import SQLModel, Field, UniqueConstraint
+from datetime import datetime
+
+class CompanyJobs(SQLModel, table=True):
+    __tablename__ = 'company_jobs'
+    job_id: int = Field(default=None, primary_key=True)
```

Comment:
This should not be null - should increment automatically 
```suggestion
    company_job_id: int = Field(default=None, primary_key=True)
```